solidity-syntax.k

```diff
@@ -340,9 +342,12 @@ module SOLIDITY-SYNTAX
                            | "throw" ";"
                            | SimpleStatement ";" [strict]
                            | "require" "(" Expression ")" ";" 
-//kan                           | "_" ";"
+                           | "_;"
```

Comment:
Any reason why this is `"_;"` and not `"_" ";"`? 
Did you have any problem with that or is it just an arbitrary choice? 

I suppose that with this definition it will not work if you have spaces between `_` and `;` in the program, right?